src/TestData/Factories/DonationFactory.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace GiveTestData\TestData\Factories;
+
+use GiveTestData\TestData\Framework\Factory;
+
+/**
+ * Class Donation
+ * @package GiveTestData\TestData\Factories
+ */
+class DonationFactory extends Factory {
+	/**
+	 * @var string
+	 */
+	private $status;
+	/**
+	 * @var int
+	 */
+	private $amount;
+
+	/**
+	 * @var string
+	 */
+	private $currency;
+
+	/**
+	 * @param string $status
+	 */
+	public function setDonationStatus( $status ) {
+		$this->status = $status;
+	}
+
+	/**
+	 * Check is valid donation status
+	 *
+	 * @param string $status
+	 *
+	 * @return bool
+	 */
+	public function checkDonationStatus( $status ) {
```

Comment:
I'm not seeing this public method used anywhere - is it being used?

Reply:
@kjohnson yes, https://github.com/impress-org/givewp-test-data/blob/0ce77a905b514e8243f9a243e2d7a498b62c6f24/src/TestData/Commands/DonationSeedCommand.php#L82